--- .venv/lib/python3.12/site-packages/django/contrib/admin/migrations/0001_initial.py	2024-11-20 14:00:55.914134287 +0800
+++ .venv/lib/python3.12/site-packages/django/contrib/admin/migrations/0001_initial.py.changed	2024-11-20 14:00:29.971062320 +0800
@@ -5,7 +5,7 @@
 
 class Migration(migrations.Migration):
     dependencies = [
-        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        # migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ("contenttypes", "__first__"),
     ]
 
@@ -52,14 +52,15 @@
                         verbose_name="content type",
                     ),
                 ),
-                (
-                    "user",
-                    models.ForeignKey(
-                        to=settings.AUTH_USER_MODEL,
-                        on_delete=models.CASCADE,
-                        verbose_name="user",
-                    ),
-                ),
+                # (
+                #     "user",
+                #     models.ForeignKey(
+                #         to=settings.AUTH_USER_MODEL,
+                #         on_delete=models.CASCADE,
+                #         verbose_name="user",
+                #     ),
+                # ),
+                ('user', models.IntegerField(verbose_name='user',)),
             ],
             options={
                 "ordering": ["-action_time"],
